crm/optimize/outlook_extension.rst

```diff
@@ -137,7 +137,27 @@ Connect to your database
 Pricing
 =======
 
-| *Partner Autocomplete* is an *In-App Purchase (IAP)* service, which requires prepaid credits to be
+The extension is **free** and does not require any purchase to be installed or to create entries in
+your database.
+
+However, this extension provides you with *Lead Enrichment* in the **Company Insights** section.
+This service is part of a paid service named **Lead Generation**.
```

Comment:
Might be a good idea to link to the articles about Lead Enrichment and/or lead generation

Reply:
Thanks for the idea.
There is no such doc but I added a link to the related Odoo Learn video in the `seealso` section.